alpha/lib/model/kuserPeer.php

```diff
@@ -89,6 +92,8 @@ public static function getActiveKuserByPartnerAndUid($partner_id , $puser_id)
 	
 	public static function createKuserForPartner($partner_id, $puser_id, $is_admin = false)
 	{
+		if(strlen($puser_id) > 100)
+			$puser_id = substr($puser_id, 0, 100);
```

Comment:
Move the 2 line logic to a static private function - 
getvalidPuserStr() 
Later, we can use this function to add more validation rules ( if we will be required) but in one place.